addons/account_edi/models/account_edi_format.py

```diff
@@ -227,7 +227,7 @@ def _cancel_payment_edi(self, payments):
     # Import methods to override based on EDI Format
     ####################################################
 
-    def _create_invoice_from_xml_tree(self, filename, tree, journal=None):
+    def _create_invoice_from_xml_tree(self, filename, tree, journal=None, move_type=None):
```

Comment:
Cannot change the signature of methods in stable

Reply:
Even with a default param ? :/